src/commands/deploy/cmd/run.ts

```diff
@@ -188,7 +188,7 @@ export async function handler(_user: TState, args: {env: string, resume: boolean
                     anvil,
                     testClient,
                     overrideEoaPk,
-                    etherscanApiKey: !args.fork,
+                    etherscanApiKey: args.fork ? false : undefined,
```

Comment:
this is the core of the fix.

if `etherscanApiKey` is false, zeus doesn't prompt. Here, I attempt to disable it fork tenderly forks by using `!args.fork`. In reality, I want _undefined_ if the fork isn't set, not false.